cmd/api/notifications/notifications.go

```diff
@@ -0,0 +1,38 @@
+package notifications
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+)
+
+type Ntfy struct {
+	baseURL string
+	enabled bool
+	client  *http.Client
+}
+
+func NewNtfy(enableNotifications bool, notificationsBaseURL string) *Ntfy {
+	return &Ntfy{
+		baseURL: notificationsBaseURL,
+		enabled: enableNotifications,
+		client:  &http.Client{},
```

Comment:
Please receive the `http.Client` as an argument.

Reply:
ok